slides/01-introduction.qmd

```diff
@@ -143,6 +143,54 @@ Many thanks to Julie Jung, Alison Hill, and Desirée De Leon for their role in c
     - Tuning hyperparameters
     - Wrapping up!
 
+## Today's data: frog hatching
```

Comment:
I kind of feel that this is an awkward place to inject a first glimpse at the data. I like the overall idea of this, but feel that it would be better if placed after the `The whole game` title slide. So you'd say something like

"we are going to broadly go over the 'whole game' of a few analyses. these analyses are going to be using this frog data and this hockey data. the general strategy for each analysis looks like this workflow image"

Reply:
You're right- moved these